Slider/Slider.less

```diff
@@ -86,52 +82,58 @@
 				display: block;
 			}
 		}
-
-		&.pressed,
-		&:focus.activateOnFocus,
-		&:focus.active {
-			.knob::before {
-				transform: translate3d(-50%, -50%, 0) scale(1);
-			}
-		}
 	});
 
 	.applySkins({
 		.progressBar {
 			background-color: @agate-slider-bg-color;
+			background-image: @agate-slider-bg-image;
 		}
 
 		.fill {
-			background-color: @agate-slider-bg-image;
+			background-color: @agate-slider-fill-bg-color;
 		}
 
 		.knob::before {
+			// Vertically and horizontally center this knob.
+			// Using this method rather than -top -left so the knob is more intuitively customizable.
+			transform: translate3d(-50%, -50%, 0) scale(0.8) skewX(@agate-slider-knob-skew);
+			border-radius: @agate-slider-knob-border-radius;
 			background-color: @agate-slider-knob-bg-color;
+			background-image: @agate-gradient;
 		}
 
 		.spottable({
 			&:focus.activateOnFocus,
 			&.active,
 			&.pressed {
 				.fill {
-					background-color: @agate-spotlight-bg-color;
+					background-color: @agate-slider-fill-bg-color;
 				}
 			}
 
 			.focus({
 				background-color: transparent;
 
 				.knob::before {
-					background-color: @agate-spotlight-focus-bg-image;
+					background-color: @agate-slider-knob-focus-bg-color;
 				}
 
 				&.active {
 					.knob::before {
-						background-color: @agate-slider-knob-bg-color;
+						background-color: @agate-slider-knob-focus-bg-color;
 						border-color: @agate-spotlight-focus-bg-image;
 					}
 				}
 			});
+
+			&.pressed,
+			&:focus.activateOnFocus,
+			&:focus.active {
+				.knob::before {
+					transform: translate3d(-50%, -50%, 0) scale(1) skewX(@agate-slider-knob-skew);
```

Comment:
Like the above comment, but a "pressed" state version of the same variable, to differentiate the `0.8` vs `1` scaling. Some themes may not want to scale the knob.